tmail-backend/integration-tests/jmap/distributed-jmap-integration-tests/src/test/java/com/linagora/tmail/james/DistributedLinagoraSecondaryBlobStoreTest.java

```diff
@@ -256,4 +267,39 @@ void sendEmailShouldResultingInEventuallySavingDataToBothObjectStoragesWhenSecon
                 assertThat(blobIds2).hasSameElementsAs(blobIds);
             });
     }
+
+    @Test
+    void sendEmailShouldResultingInEventDeadLettersWhenSecondStorageIsDownForLongTime(GuiceJamesServer server) throws Exception {
+        secondaryS3.pause();
+
+        given()
+            .body(LinagoraEmailSendMethodContract$.MODULE$.bobSendsAMailToAndre(server))
+            .when()
+            .post()
+            .then()
+            .statusCode(HttpStatus.SC_OK)
+            .contentType(JSON)
+            .extract()
+            .body()
+            .asString();
+
+        Thread.sleep(ONE_MINUTE);
```

Comment:
too long for test?

Reply:
is it necessary? I see `calmlyAwait.atMost(ONE_MINUTE)` above

Reply:
reduce calmlyAwait.atMost(ONE_MINUTE) to ten seconds

Reply:
```suggestion
   calmlyAwait.atMost(ONE_MINUTE).untilAsserted(() -> {
       assertThat(blobStoreProbe.getEventDeadLetters().failedIds(group)
                    .collectList().block())
                    .isNotEmpty();
   };
```

Reply:
I see it is still ONE_MINUTE. Can we switch to await?

Reply:
await for what?